### PR TITLE
Internal improvement: Update compile-solidity tests for 0.8.0

### PR DESCRIPTION
**Pull request by haltman-at**
Updates the compile-solidity tests that always use the latest version so they'll work with 0.8.0 and the build won't break!

### CODE REVIEW

File: packages/compile-solidity/test/sources/v0.8.x/Version7Pragma.sol [hunk 1]
```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.8.0;
+
+contract Version7Pragma {
```
Comment (eggplantzzz) [inline packages/compile-solidity/test/sources/v0.8.x/Version7Pragma.sol:3]: Should this be `Version8Pragma` along with a file rename?

Comment (haltman-at) [reply packages/compile-solidity/test/sources/v0.8.x/Version7Pragma.sol:3]: Oops! Thank you.

**Review by eggplantzzz — APPROVED** [review]
I like it!

**Comment by haltman-at** [comment]
Thanks to @eggplantzzz for figuring out the correct solution here!  (Upgrading Ubuntu rather than doing manual package management like I was trying.)

**Comment by haltman-at** [comment]
(Apologies about the messy commits here -- I'd squash it all, but I want to get this merged as soon as possible.)

**Pull request merged**